clients/extension/src/sessions/mutations/useClearVaultSessionsMutation.ts

```
@@ -0,0 +1,31 @@
+import { useInvalidateQueries } from '@lib/ui/query/hooks/useInvalidateQueries'
+import { UseMutationOptions, useMutation } from '@tanstack/react-query'
+import {
+  appSessionsQueryKey,
+  useVaultsAppSessionsMutation,
+  useVaultsAppSessionsQuery,
+} from '../state/appSessions'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)